float32[] jacobianEE_row1
float32[] jacobianEE_row2
float32[] jacobianEE_row3
float32[] jacobian0_row1
float32[] jacobian0_row2
float32[] jacobian0_row3
float32[] endpoint_row1
float32[] endpoint_row2
float32[] endpoint_row3
float32[] rotation_row1
float32[] rotation_row2
float32[] rotation_row3
